src/main/java/org/jenkinsci/plugins/tokenmacro/MacroEvaluationException.java

```diff
@@ -30,7 +32,26 @@
  * @author Kohsuke Kawaguchi
  */
 public class MacroEvaluationException extends Exception {
+    
+    private final @CheckForNull String macroName;
+    
     public MacroEvaluationException(String message) {
+        this(message, null);
```

Comment:
I think the parameters are backwards here. Based on the constructor below, I believe it should be this(null, message)